litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/index.tsx

```diff
@@ -7,25 +7,30 @@ interface LinearProgressBarProps {
   maxValue: number;
   isInTable: boolean;
 }
-
 const AnalyticsLinearProgressBar: React.FC<LinearProgressBarProps> = ({
   value,
   maxValue,
   isInTable,
-}) => {
-  const width: number = 2;
+}) => 
+{
   const resultValue = ((value as number) / (maxValue as number)) * 100;
-
+  const defaultSize = 20;
   const { palette } = useTheme();
 
   return (
     <Line
       percent={resultValue}
       strokeWidth={width}
       trailWidth={width}
-      strokeColor={isInTable ? palette.primary.dark : palette.secondary.dark}
+      strokeColor={
+        isDefault
+          ? theme.palette.secondary.dark
+          : resultValue > 30 && resultValue <= 60
+          ? theme.palette.warning.main
+          : resultValue > 60
+          ? theme.palette.primary.dark
+          : theme.palette.error.dark
+      }
```

Comment:
This change does not satisfy our requirement of changing `Line` to `ProgressBar`